renderer/style.css

```diff
@@ -21,3 +22,35 @@ body {
 .file-picker span {
   pointer-events: none;
 }
+
+/* Files modal (when some files have been selected) */
+.file-modal {
+  display: grid;
```

Comment:
Grid? Yuck xD

Reply:
Seriously? Flex and grid are the only reasons I can get things to align properly in CSS :P